crates/ra_syntax/src/string_lexing.rs

```diff
@@ -172,6 +240,51 @@ impl<'a> Parser<'a> {
             ))
         }
     }
+
+    pub fn parse_ignore_newline(&mut self, start: TextUnit) -> Option<StringComponent> {
+        // In string literals, when a `\` occurs immediately before the newline, the `\`,
+        // the newline, and all whitespace at the beginning of the next line are ignored
+        match self.peek() {
+            Some('\n') | Some('\r') => {
+                self.skip_whitespace();
```

Comment:
Should this eat a string of the form
```rust
"\
<all whitespace>
more text"
```
where there is an entirely whitespace line. See that `\n` is whitespace [playground link](https://play.rust-lang.org/?version=stable&mode=debug&edition=2015&gist=49a54d3f88d6d6a06ac999a8728217cd).

The current behaviour appears to be that, unless I'm mistaken.

Reply:
It will indeed eat all the whitespace, including new lines, as does the Rust compiler. See [this playground link](https://play.rust-lang.org/?version=stable&mode=debug&edition=2015&gist=fa72acf1666afe79e6201b413b78fd13)